modules/network-observability-operator-uninstall.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+
+// * installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-operator-uninstall_{context}"]
+
+= Uninstalling the Network Observability operator
+You can install the Network Observability operator using the {product-title} web console Operator Hub.
+ 
+.. In the {product-title} web console, click *Operators* -> *Installed Operators*.
+
+.. Click the options menu {kebab} next to the  *Network Observability operator* and select *Uninstall Operator*.
```

Comment:
Is this really it?  There's nothing that needs to be done about removing the flow collector?  
Should we add anything here about Loki?  I know we need to be careful if it's used for other projects besides us -- but wondering if it's at least worth a mention ....

Reply:
ACK..I do think this is incomplete, per a Slack convo with @stleerh. He says uninstallation is pretty gnarly 😬 so yea I think there's more steps here I need to know about.

Reply:
Note this is for uninstalling upstream operator, haven't tried these steps for downstream

Reply:
@stleerh Did you say you had some uninstall guidance you could send my way?

Reply:
PS: we should also mention uninstalling Loki and Kafka (if it was installed), but since we don't enforce a single way to install them we cannot be comprehensive. I would simply mention to refer to the uninstallation process that corresponds to however they did the installation

Reply:
Ok, addressed. I added a NOTE to the Loki Install, which details the following:
refer to the uninstallation process that corresponds with the method you used to install Loki. You might have remaining `ClusterRoles` and `ClusterRoleBindings` or data stored in object store and PVs that must be removed